### PR TITLE
Assignment 6

### PR DESCRIPTION
**Pull request by omerana14**

### CODE REVIEW

**Review by MKadeer17 — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

File: Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                let f = fact(FactNumber[i]);
+                factorial.unshift(f);
+            }
+
+            function fact(n) {
+                let f = n;
+                for (let i = 1; i < n; i++) f = f * i;
+                return f;
+            }
+            document.write(`Factorial in descending Order :${factorial}`);
```
Comment (Snehal-113) [inline Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: factorial must be printed in descending order,use reverse method

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html (hunk 1 above)
Comment (MKadeer17) [reply Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: Like this.
```suggestion
            document.write(`Factorial in descending Order :${factorial.sort().reverse();}`);
```

Comment (omerana14) [reply Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: Thank you so much Qadir i ll implement this.

Comment (omerana14) [reply Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: Done, please check.

Comment (MKadeer17) [reply Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: Welcome

File: Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html [hunk 2]
```diff
@@ -59,7 +59,9 @@ <h4>Check factorial for each number :</h4>
                 for (let i = 1; i < n; i++) f = f * i;
                 return f;
             }
-            document.write(`Factorial in descending Order :${factorial}`);
+            // Check Factorial in asending order
+            // document.write(`Factorial in Asending Order :${factorial}<br>`); 
+            document.write(`Factorial in descending Order :${factorial.reverse()}`);
```
Comment (Snehal-113) [inline Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: omkar reverse will only reverse the array elements...u need to sort it first n then reverse it..
https://www.w3schools.com/jsref/jsref_sort.asp

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html (hunk 2 above)
Comment (omerana14) [reply Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html]: changes done mam please check

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**